src/components/Header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import s from "./Header.module.css";
```

Comment:
"s" is hard to read in terms of understanding what this variable is for.
Consider using a full name such as "styles".

Don't forget to do this in all other file of the project.

Reply:
"s" is used for shorter spelling in tags. will be fixed in the future

Reply:
Fine